metadata.json

```diff
@@ -1,23 +1,79 @@
 {
-  "name": "mrpuppet-tagmail",
-  "version": "0.1.3",
-  "author": "mrpuppet",
-  "summary": "A report processor plugin for re-enabling the tagmail feature in PE 3.7 onwards. Use this to have Puppet send emails based on tags",
+  "name": "puppetlabs-tagmail",
+  "version": "1.0.0",
```

Comment:
This hasn't gone through QA yet. Should we do something other than a 1.0.0 first until it gets reviewed?